src/app/pluginAdapter.test.js

```diff
@@ -98,4 +122,40 @@ describe("app/pluginAdapter", () => {
       );
     });
   });
+
+  describe("findNodeType", () => {
+    // No need to check that it erros when there is not a unique type match,
```

Comment:
I disagree in principle with the claim that “the implementation is
correct, so there’s no need to test it”. The implementation is of course
correct now, but I’m free to change the implementation so that it no
longer calls `findUniqueMatch`, and all that.

With the above in mind, if you still don’t want to test this error case,
I’m willing to let it slide.